CHANGES.md

```diff
@@ -9,9 +9,10 @@
 ### Bug Fixes
 
 * filter: Fixed unintended behavior in which grouping by `day` would "work" when used with `month` and/or `year`. Updated so it will be ignored. [#1070][] (@victorlin)
+* filter: Fixed unintended behavior in which grouping by `month` with ambiguous years would "work". Updated so date ambiguity is checked properly for all generated columns. [#1072][] (@victorlin)
 
 [#1067]: https://github.com/nextstrain/augur/pull/1067
-[#1070]: https://github.com/nextstrain/augur/pull/1070
+[#1072]: https://github.com/nextstrain/augur/pull/1072
```

Comment:
The #1070 reference is replaced here, but appears to be still in use…

Reply:
Oops, fixed: 6ec01057b14c4cedcb68b189c554e974df610c58